apibinding/apibinding/api.py

```
@@ -53,16 +53,101 @@ def _get_response(self, ret_uuid):
+            # Fallback if format fails
+            msg_parts.append("[code: <format error>, description: <format error>, details: <format error>]")
+
+        # Safely handle elaboration with proper error handling
+        if elaboration is not None:
+            try:
+                elaboration_str = str(elaboration).strip()
+                if elaboration_str:
+                    msg_parts.append(", elaboration: \n{0}".format(elaboration_str))
+            except Exception:
+                # If str() fails, skip elaboration rather than crashing
+                pass
```

Ruff (0.14.4)

`try`-`except`-`pass` detected, consider logging the exception

(S110)